### PR TITLE
LIT 6960 to 66

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Some new qenes from Abb. 57.

### CODE REVIEW

File: new/LIT6958Egziamlakfatari.xml [hunk 1]
```diff
@@ -32,8 +32,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <creation/>
             <abstract>
-                <p>This poem is referred to as qǝne of the type <foreign xml:lang="gez">kǝbr yǝʾǝti</foreign>,
-                    in <ref type="mss" corresp="ESdd037"/>, and attributed to <persName ref="PRS14409Yohannes"/>.</p>
+                <p>This poem is referred to as qǝne, in <ref type="mss" corresp="BNFabb57"/>, attributed to <persName ref="PRS14051LaekaMaryam"/>, 
+                    possibly of the type <foreign xml:lang="gez">ʿǝṭāna mogar</foreign>.</p>
```
Comment (eu-genia) [inline new/LIT6958Egziamlakfatari.xml:36]: use now `<term key="EtanaMogar">ʿǝṭāna mogar</term>`

File: new/LIT6958Egziamlakfatari.xml [hunk 2]
```diff
@@ -32,8 +32,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <creation/>
             <abstract>
-                <p>This poem is referred to as qǝne of the type <foreign xml:lang="gez">kǝbr yǝʾǝti</foreign>,
-                    in <ref type="mss" corresp="ESdd037"/>, and attributed to <persName ref="PRS14409Yohannes"/>.</p>
+                <p>This poem is referred to as qǝne, in <ref type="mss" corresp="BNFabb57"/>, attributed to <persName ref="PRS14051LaekaMaryam"/>,
```
Comment (eu-genia) [inline new/LIT6958Egziamlakfatari.xml:35]: `<term key="Qene">qǝne</term>`

Comment (eu-genia) [reply new/LIT6958Egziamlakfatari.xml:35]: here and elsewhere

File: new/LIT6958Egziamlakfatari.xml (hunk 1 above)
Comment (eu-genia) [reply new/LIT6958Egziamlakfatari.xml:36]: here and elsewhere

File: new/LIT6960Emmahtsene.xml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation notBefore="1580" notAfter="1610"/>        
+            <abstract>
+                <p>The poem is referred to as qǝne of the type <foreign xml:lang="gez">ʿǝṭāna mogar</foreign>, in <ref type="mss" corresp="BNFabb57"/>, and attributed to
```
Comment (eu-genia) [inline new/LIT6960Emmahtsene.xml:37]: term key

File: new/LIT6961Emnamalakot.xml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation notBefore="1580" notAfter="1610"/>
+            <abstract>
+                <p>The poem, known from <ref type="mss" corresp="BNFabb57"/>,
+                    may be of the type <foreign xml:lang="gez">ʿǝṭāna
+                        mogar</foreign> and attributed to <persName ref="PRS14052SardaMaryam"/>.</p>
```
Comment (eu-genia) [inline new/LIT6961Emnamalakot.xml:37]: term

File: new/LIT6962Taramemnu.xml [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>       
+            <creation notBefore="1580" notAfter="1610"/>
+            <abstract>
+                <p>The poem, known from <ref type="mss" corresp="BNFabb57"/>, may be of the type
+                        <foreign xml:lang="gez">ʿǝṭāna mogar</foreign> and attributed to <persName
```
Comment (eu-genia) [inline new/LIT6962Taramemnu.xml:37]: term

File: new/LIT6964Liqakahenat.xml [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation notBefore="1580" notAfter="1610"/>
+            <abstract>
+                <p>The poem attested in <ref type="mss" corresp="BNFabb57"/>, is most probably a qǝne
+                    of the type <foreign xml:lang="gez">ʿǝṭāna mogar</foreign> (the indication is erased and only partly readable). The poem
```
Comment (eu-genia) [inline new/LIT6964Liqakahenat.xml:36]: term

File: new/LIT6965Egziabfatare.xml [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>This poem is referred to as qǝne of the type <foreign xml:lang="gez">kǝbr yǝʾǝti</foreign>,
```
Comment (eu-genia) [inline new/LIT6965Egziabfatare.xml:35]: term

File: new/LIT6966Bahagara.xml [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation notBefore="1580" notAfter="1610"/>
+            <abstract>
+                <p>The poem, attested in <ref type="mss" corresp="BNFabb57"/>, is a qǝne
+                    of the type <foreign xml:lang="gez">za-yǝʾǝze</foreign>, assigned to <persName ref="PRS14051LaekaMaryam"/>.</p>
```
Comment (eu-genia) [inline new/LIT6966Bahagara.xml:36]: term

**Review by eu-genia — CHANGES_REQUESTED** [review]
please assign type keywords here and elsewhere

File: new/LIT6958Egziamlakfatari.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/LIT6958Egziamlakfatari.xml:36]: ok

File: new/LIT6958Egziamlakfatari.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/LIT6958Egziamlakfatari.xml]: ok

File: new/LIT6960Emmahtsene.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/LIT6960Emmahtsene.xml]: ok

File: new/LIT6962Taramemnu.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply new/LIT6962Taramemnu.xml:37]: ok

File: new/LIT6964Liqakahenat.xml (hunk 6 above)
Comment (DenisNosnitsin1970) [reply new/LIT6964Liqakahenat.xml:36]: ok

File: new/LIT6965Egziabfatare.xml (hunk 7 above)
Comment (DenisNosnitsin1970) [reply new/LIT6965Egziabfatare.xml]: ok

File: new/LIT6966Bahagara.xml (hunk 8 above)
Comment (DenisNosnitsin1970) [reply new/LIT6966Bahagara.xml:36]: ok

File: new/LIT6951Bahaki.xml [hunk 9]
```diff
@@ -32,7 +32,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <creation notBefore="1700" notAfter="1900"/>
             <abstract>
-                <p>This poem is referred to as qǝne of the type <foreign xml:lang="gez">mawaddǝs</foreign>,
+                <p>This poem is referred to as <term key="Qene">qǝne</term> of the type <foreign xml:lang="gez">mawaddǝs</foreign>,
```
Comment (eu-genia) [inline new/LIT6951Bahaki.xml:35]: mawaddes is also a keyword

File: new/LIT6958Egziamlakfatari.xml (hunk 1 above)
Comment (eu-genia) [reply new/LIT6958Egziamlakfatari.xml:36]: not yet?

File: new/LIT6949Kwellu.xml [hunk 10]
```diff
@@ -31,7 +31,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <creation/>
             <abstract>
-                <p>This qǝne poem is marked as being of the type <foreign xml:lang="gez">kʷǝllǝkǝmu gǝʿǝz</foreign> 
+                <p>This <term key="Qene">qǝne</term> poem is marked as being of the type <foreign xml:lang="gez">kʷǝllǝkǝmu gǝʿǝz</foreign> 
                     in <ref type="mss" corresp="DMGYi135IIIC7"/> and of the type <foreign xml:lang="gez">mawaddǝs</foreign>
```
Comment (eu-genia) [inline new/LIT6949Kwellu.xml:35]: Mawaddes is a keyword

File: new/LIT6949Kwellu.xml [hunk 11]
```diff
@@ -31,7 +31,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <creation/>
             <abstract>
-                <p>This qǝne poem is marked as being of the type <foreign xml:lang="gez">kʷǝllǝkǝmu gǝʿǝz</foreign> 
+                <p>This <term key="Qene">qǝne</term> poem is marked as being of the type <foreign xml:lang="gez">kʷǝllǝkǝmu gǝʿǝz</foreign>
```
Comment (eu-genia) [inline new/LIT6949Kwellu.xml:34]: Kwellekemu is a keyword

File: new/LIT6955Lamangesta.xml [hunk 12]
```diff
@@ -32,7 +32,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <creation notBefore="1700" notAfter="1900"/>
             <abstract>
-                <p>This poem is referred to as qǝne of the type <foreign xml:lang="gez">ʿǝṭāna mogar</foreign>,
+                <p>This poem is referred to as <term key="Qene">qǝne</term> of the type <foreign xml:lang="gez">ʿǝṭāna mogar</foreign>,
```
Comment (eu-genia) [inline new/LIT6955Lamangesta.xml:35]: EtanaMogar

File: new/LIT6965Egziabfatare.xml [hunk 13]
```diff
@@ -32,13 +32,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
-                <p>This poem is referred to as qǝne of the type <foreign xml:lang="gez">kǝbr yǝʾǝti</foreign>,
+                <p>This poem is referred to as a <term key="Qene"/> of the type <foreign xml:lang="gez">kǝbr yǝʾǝti</foreign>,
                     in <ref type="mss" corresp="ESdd037"/>, and attributed to <persName ref="PRS14409Yohannes"/>.</p>
             </abstract>
             <textClass>
                 <keywords>
                     <term key="ChristianLiterature"/>
                     <term key="Prayers"/>
+                    <term key="Qene"/>
+                    <term key="EtanaMogar">ʿǝṭāna mogar</term>
```
Comment (eu-genia) [inline new/LIT6965Egziabfatare.xml:43]: not EtanaMogar, KebrYeeti? please provide suitable keywords, also in the text

File: new/LIT6966Bahagara.xml (hunk 8 above)
Comment (eu-genia) [reply new/LIT6966Bahagara.xml:36]: Keyword for the type, see https://github.com/BetaMasaheft/Documentation/issues/2504

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT6965Egziabfatare.xml (hunk 13 above)
Comment (DenisNosnitsin1970) [reply new/LIT6965Egziabfatare.xml:43]: done

File: new/LIT6955Lamangesta.xml (hunk 12 above)
Comment (DenisNosnitsin1970) [reply new/LIT6955Lamangesta.xml:35]: ok

**Review by eu-genia — APPROVED** [review]
OK, 

Alternatively you can also use keywords in the text, then they do not have to be repeated, 

`This poem is referred to as <term key="Qene">qǝne</term> of the type <term key="EtanaMogar">ʿǝṭāna mogar</term>` 

etc

**Pull request merged**